OnOffExample/OnOffExample.csproj

```diff
@@ -10,7 +10,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>

```

Comment:
The examples will need to move to project.json in order to use the version of the code that's built within the solution rather than the NUPKG from NuGet, IIRC.